web/components/video/VideoJS/VideoJS.tsx

```diff
@@ -36,14 +36,14 @@ export const VideoJS: FC<VideoJSProps> = ({ options, onReady }) => {
     ) {
       (
         videojs.getPlayer(videoRef.current).tech({ IWillNotUseThisInPlugins: true }) as any
-      ).vhs.xhr.beforeRequest = o => {
+      ).vhs.xhr.onRequest = (xhr, o) => {
```

Comment:
The code this is referencing is here:

https://github.com/videojs/http-streaming/blob/8456cb3c1c8af207156c0330a9cc2050592e8880/src/videojs-http-streaming.js#L558-L560

Replacing this wouldn't work. This callback needs to be passed as a parameter to the onRequest call instead of replacing it.